src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -180,8 +180,8 @@ internal BoundExpression ConvertPatternExpression(TypeSymbol inputType, CSharpSy
                 {
                     case ConversionKind.ExplicitDynamic:
                     case ConversionKind.ImplicitDynamic:
-                        // Since the input was `dynamic`, which is equivalent to `object`, there must also
-                        // exist some unboxing, identity, or reference conversion as well, making the conversion legal.
+                    // Since the input was `dynamic`, which is equivalent to `object`, there must also
```

Comment:
>// Since the input was `dynamic`, which is equivalent to `object`, there must also [](start = 20, length = 82)

Please revert this file to its original state.